types/react-native-community__cli/react-native-community__cli-tests.ts

```diff
@@ -0,0 +1,15 @@
+import { UserDependencyConfigT } from 'react-native-community__cli';
+const pluginConfig: UserDependencyConfigT = {
```

Comment:
I'm not sure I follow how the library is used. The `d.ts` in this PR is basically just exporting interfaces, but there are no declared values. Also, when I try to dump the contents of the module, I get something completely different:

```
> require('@react-native-community/cli')
{ run: [AsyncFunction: run],
  init: [AsyncFunction: initCompat],
  loadConfig: [Function: loadConfig] }
```

Reply:
@PranavSenthilnathan thank you very much for your review. I added the types of these 3 functions